### ISSUE TITLE
Server Logs (Player Left)

### ISSUE THREAD

**Issue by Gravxd** [citizenfx/txAdmin#270]
I've experienced this bug on multiple servers so its not just one installation of tx. When using the side panel on the right for players, it will show their full id's like steam & license etc. But if you use server logs and click their name within the serverlogs, it will only give you the license. To get full identifiers, i inspect element the player's name in server logs and it gives me their steam id + sometimes discord id if its linked.

Just wondering is this intentional or is it a bug as it's not been changed over the new updates.

TLDR: Full identifiers do not display in server logs as they do in the player bar (right side)

**Comment by Arm-0001** [comment]
Pretty sure this is intentional as offline players only get their license showed

**Comment by Gravxd** [comment]
> Pretty sure this is intentional as offline players only get their license showed

This applies to online players too in some cases when using server logs

**Comment by Gravxd** [comment]
Additionally, if you inspect, you can see their full id's so i wouldn't see the harm in showing the full id's as they are online in some cases anyway & visible in other tabs like the player bar

**Comment by Arm-0001** [comment]
Wel I'll wait for tabarra for a full answer but it's either one of the following 
1. Didn't code it yet
2. Didn't think it's needed

**Comment by rpiadam** [comment]
only issue whem dc server doesnt save last spawn location players leave like 5 mins behind

**Comment by wowjeeez** [comment]
Can you please stop posting duplicate and non-pr related questions?

**Comment by TasoOneAsia** [comment]
Unable to reproduce and stale issue. If this is still occurring reopen with further details.

**Comment by tabarra** [comment]
For the sake of tracking: this will be totally reworked anyways.  
In the next update you will be able to see full identifiers of anyone connected since txAdmin started (doesn't matter if server restarted or not), and there will be persistent log files containing player/server id, and you can just find the moment he logged since it will contain all the identifiers.
